deploy/10_4_deploy_gold_gbp_price_feeds.ts

```diff
@@ -0,0 +1,124 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployFunction } from "hardhat-deploy/types";
+import * as constants from "../utils/Constants";
+import {
+  formatAddress,
+  getBDStableWbtcPool,
+  getBDStableWethPool,
+  getBgbp,
+  getBot,
+  getBxau,
+  getDeployer,
+  getWethPairOracle
+} from "../utils/DeployedContractsHelpers";
+import type { DeployResult } from "hardhat-deploy/dist/types";
+import { to_d12 } from "../utils/NumbersHelpers";
+
+export const PriceFeedContractNames = {
```

Comment:
It's not a good place to export from. Numerous times I wanted to temporarily delete deployment files to do some experiments and the whole project exploded

Reply:
moved it to constants, also changed all the places we were using the `PriceFeedContractNames` from `7_deploy_price_feeds.ts` but didn't remove the definition for this file since it's a deployment script we already executed on RSK so I prefer not to change it (just removed the export their)